python/orca/src/bigdl/orca/data/utils.py

```diff
@@ -467,7 +467,22 @@ def check_col_exists(df, columns):
                           str(col_not_exist) + " do not exist in this Table")
 
 
-def transform_to_shard_dict(data, featureCols, labelCol):
+def shards_pd_df_to_shards_dic(data, featureCols, labelCol):
```

Comment:
The name is still very generic; what does this mean exactly? A user will not be able to understand what `shards_pd_df_to_shards_dic` is at all.

What are the valid types of each column? Float or array or ...?

Do you assembly these columns into a vector or a tensor or ...? If so, make it explicit in the name.